ts/features/walletV3/onboarding/screens/WalletOnboardingSelectPaymentMethodScreen.tsx

```diff
@@ -62,7 +62,7 @@ const WalletOnboardingSelectPaymentMethodScreen = () => {
     dispatch(walletGetPaymentMethods.request());
   });
 
-  const handleSelectedPaymentMethod = (
+  const handleSelectedPaymentMethod = async (
```

Comment:
same as `434`, why is this async?

Reply:
Ops! You're right! 

Addressed into ff5433899a3d94f3b68bee9bf732ba19a10238c6